Code/GraphMol/molopstest.cpp

```diff
@@ -1976,6 +1976,100 @@ void testAddHsCoords() {
   delete m;
   delete m2;
 
+  // make sure Hs are on the xy plane if conformer is 2D
```

Comment:
This is ok as-is, but I've started (mostly) adding new test cases so that more "modern" style (like using `_smiles`) can be used.